pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -97,9 +98,24 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 		// (e.g., allowedTopologies, volumeBindingMode), and hence may become
 		// schedulable upon StorageClass Add or Update events.
 		{Event: framework.ClusterEvent{Resource: framework.StorageClass, ActionType: framework.Add | framework.Update}},
+
 		// We bind PVCs with PVs, so any changes may make the pods schedulable.
-		{Event: framework.ClusterEvent{Resource: framework.PersistentVolumeClaim, ActionType: framework.Add | framework.Update}},
+		// QHintFn is added only for PersistentVolumeClaim because:
```

Comment:
Is it really "only" pvc? I haven't looked at @YamasouA's PRs yet though, they look alrighty; no reference to other resources.

- https://github.com/kubernetes/kubernetes/pull/125171
- https://github.com/kubernetes/kubernetes/pull/125097

Reply:
@sanposhiho CSIDriver and CSINode are resources that rarely get updated.
If adding QHints for StorageClass and CSIStorageCapacity is unnecessary, it seems that adding QHints for CSIDriver and CSINode would also be unnecessary.
If QHints are to be added for CSIDriver and CSINode, it might be more consistent to also add QHints for StorageClass and CSIStorageCapacity.

@carlory I would be very interested to hear your perspective on this issue, if you have a moment to share your thoughts.

Reply:
> CSIDriver and CSINode are resources that rarely get updated.

I cannot say about it. It depends on users. 

> If QHints are to be added for CSIDriver and CSINode, it might be more consistent to also add QHints for StorageClass and CSIStorageCapacity.

If the hint func is simple enough, it's better to add it. If it takes a lot of time, it's better to skip it.

Reply:
> If the hint func is simple enough, it's better to add it. If it takes a lot of time, it's better to skip it.

This ^. 
So, the point is whether we can implement QHint _simply_, not how often QHint is supposed to get called.

And, the _current_ definition of simplicity here is that whether we can implement QHint with added/updated/deleted resource (= PVC in this PR) and the target Pod. That's why I said -
> they look alrighty; no reference to other resources.